_tests/custom_tests/tests/__snapshots__/EnvConfigTest__test_config_with_local_paths__1.txt

```diff
@@ -15,7 +15,7 @@
         {
             "slug": "fake-plugin-ENV_ID_NORMALIZED",
             "entrypoint": null,
-            "source": "\/tmp\/fake-plugin-ENV_ID_NORMALIZED.zip",
+            "source": "\/var\/folders\/b4\/mftz73hs2mj70_7qjsrbq7lw0000gn\/T\/fake-plugin-ENV_ID_NORMALIZED.zip",
```

Comment:
@Luc45 Looks like this should have been normalized right?

Reply:
Yup, it should. It's fine to change the snapshot manually for now and we can look into it later